modules/structs/repo_commit.go

```diff
@@ -20,6 +20,8 @@ type Identity struct {
 type CommitMeta struct {
 	URL string `json:"url"`
 	SHA string `json:"sha"`
+	// swagger:strfmt date-time
+	Timestamp time.Time `json:"timestamp"`
```

Comment:
one thing: the name

should we realy call it "timestamp" aond not "created" ?

Reply:
Looking at the GitHub API specifications, they usually use "date" as the field for dates, but to me created makes more sense. I'll change it!